osu.Game.Rulesets.Taiko/Mods/TaikoModFlashlight.cs

```diff
@@ -63,7 +63,7 @@ public TaikoFlashlight(TaikoModFlashlight modFlashlight, TaikoPlayfield taikoPla
             /// </param>
             private Vector2 adjustSizeForPlayfieldAspectRatio(float size)
             {
-                return new Vector2(0, size * taikoPlayfield.DrawHeight / TaikoPlayfield.DEFAULT_HEIGHT);
+                return new Vector2(0, size * taikoPlayfield.Parent!.Scale.Y);
```

Comment:
Can you give insight into this change? It's in a commit which claims to be refactoring the usage of `DEFAULT_HEIGHT`, but this changes the size of the flashlight and is mentioned nowhere:

| Before | After |
| :---: | :---: |
| ![CleanShot 2024-01-22 at 09 24 12](https://github.com/ppy/osu/assets/191335/b61e9347-4281-4990-bd09-2e6690e7db98) | ![CleanShot 2024-01-22 at 09 23 28](https://github.com/ppy/osu/assets/191335/673337e6-c3a3-47ec-a954-aa99bc31fbb9) |

If we're looking to keep parity with master, then restoring the previous code does seem to work:

```suggestion
                return new Vector2(0, size * taikoPlayfield.DrawHeight / TaikoPlayfield.DEFAULT_HEIGHT);
```

Reply:
Hmm, this is pretty odd for two reasons:
 1. `Playfield.DrawHeight / DEFAULT_HEIGHT` cannot produce any value other than `1` with this PR, because the height of the playfield is constantly set to 400px, and the scale applied by the adjustment container is not reflected in the `DrawHeight` property as far as I understand.
 2. `Playfield.Parent!.Scale.Y` in this PR should be equal to the same value as `Playfield.DrawHeight / DEFAULT_HEIGHT` in master.

I'll double-check and see what's up.

Reply:
I checked enough times both on the values and visually as well (there is technically a difference in the value but it's too tiny to mention):

https://github.com/ppy/osu/assets/22781491/3548e7e0-3320-4e5c-8b27-ebe6994253ac

If you can still see the discrepancy, open the draw visualiser on the flashlight drawable and report the `FlashlightSize` between master/release and this PR (as well as the value of `TaikoPlayfield.DrawHeight` on master and the value of `TaikoPlayfieldAdjustmentContainer.Scale` on this PR).

Reply:
On a second check it looks okay. I think something was up with my hot reloading..